features/logging/logging_acceptance.feature

```diff
@@ -51,6 +51,7 @@ Feature: Logging smoke test case
     # Console Dashboard
     When I run the :goto_monitoring_db_cluster_logging web action
     Then the step should succeed
+    Given 60 seconds have passed
```

Comment:
@kabirbhartiRH hard-coded wait time is generally frown upon....is there another indicator that you can query to indicate things are ready to be used?

Reply:
@kabirbhartiRH how about using below steps?
```
Give I wait up to 300 seconds for the steps to pass:
"""
    When I perform the :check_monitoring_dashboard_card web action with:
      | card_name | Elastic Cluster Status |
    Then the step should succeed
"""
```
then remove `Elastic Cluster Status` from line 55(it's optional).

Reply:
@QiaolingTang @anpingli Thank you! I was trying to fix the hard coded value but since we have already have a block, I couldn't use the wait for steps to pass instruction. I will make the changes today.